src/bin/keygen.rs

```diff
@@ -1,14 +1,49 @@
+extern crate clap;
 extern crate ring;
 extern crate serde_json;
 
+use clap::{App, Arg};
 use ring::rand::SystemRandom;
 use ring::signature::Ed25519KeyPair;
+use std::env;
 use std::error;
+use std::fs::{self, File};
+use std::io::Write;
+use std::path::Path;
 
 fn main() -> Result<(), Box<error::Error>> {
+    let matches = App::new("solana-genesis")
```

Comment:
is this name supposed to be solana-keypair?

Reply:
solana-keygen, yes. fixed